app/models/user.rb

```diff
@@ -1,2 +1,3 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
```

Comment:
- [ ] A user also has an association with comments and likes, Please make sure to fix this issue so associations between models work as expected.
  - You can consider using this code.
```ruby
 has_many :comments
 has_many :likes